### PR TITLE
docs: computed columns

### PR DESCRIPTION
**Pull request by soyeric128**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

added doc: computed columns

Closes #issue

### CODE REVIEW

**Comment by sundy-li** [comment]
The feature is reverted, so the docs could be later.

**Comment by wubx** [comment]
Computed Columns need mark support version

**Comment by BohuTANG** [comment]
This work can now be continued since the #11590  merged.

**Comment by soyeric128** [comment]
> Computed Columns need mark support version

Marked as v1.1.56

File: docs/doc/14-sql-commands/00-ddl/20-table/10-ddl-create-table.md [hunk 1]
```diff
@@ -231,6 +236,53 @@ SELECT * FROM t_default_value;
...
+```
+
+The following is an example of creating a stored computed column: Whenever the values of the "price" or "quantity" columns are updated, the "total_price" column will automatically recalculate and update its stored value.
+
+```sql
+CREATE TABLE IF NOT EXISTS products (
+  id INT,
+  price FLOAT,
+  quantity INT,
+  total_price FLOAT AS (price * quantity) STORED
```
Comment (b41sh) [inline docs/doc/14-sql-commands/00-ddl/20-table/10-ddl-create-table.md:261]: total_price should use `FLOAT64` type.

**Review by b41sh — APPROVED** [review]

**Pull request merged**